spring-boot-logging-log4j2/src/test/java/com/baeldung/springbootlogging/SpringBootLoggingApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.springbootlogging;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class SpringBootLoggingApplicationTests {
```

Comment:
Integration test class names should end with "IntegrationTest" as per convention.

Reply:
Those are the default test classes generated by Spring Initializr. I've not added anything, we could delete them because useless or leave them to match what is generated by start.spring.io, but not renaming them, it would be unnecessary misleading IMHO